docs/documentation/quartz-3.x/tutorial/schedule-builder.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Tuning the Scheduler
+---
+
+# Tuning the Scheduler
+
+| Property | |
+|---|--|
+| Scheduler Id | .. |
+| Scheduler Name | .. |
+| Max Batch Size | .. |
+| InterruptJobsOnShutdown | .. |
+| InterruptJobsOnShutdownWithWait| .. |
+| BatchTriggerAcquisitionFireAheadTimeWindow | .. |
```

Comment:
@lahma if you can give me some ideas on the difference between `ScheduleId` and `SchedulerName`

Are the interrupt's mutually exclusive?

Any other commentary I can add here?

Reply:
I for one always confuse scheduler id vs name, so id is `instanceId` and name is `instanceName` in this context:

https://www.quartz-scheduler.net/documentation/quartz-3.x/configuration/reference.html#quartz-scheduler-instancename

instance name is important for the cluster - it's clustering key. instance id can be automatically generated or something like computer domain name - a node.